### ISSUE TITLE
Background problem

### ISSUE THREAD

**Issue by AlexTS1980** [wkentaro/pytorch-fcn#132]
I didn't have this probelm with FCN in caffe: background always dominates. Did I normalize images incorrectly? 

`t=transforms.Compose([transforms.ToTensor(), transforms.Normalize(mean=[0.406, 0.456, 0.486], std=[0.225, 0.224, 0.229])])
`

This normalization should correspond to an RGB image 

```
im = t(PILImage.open('dogcat1.jpg').convert("RGB")).unsqueeze(0)   
output = model(im)
plt.imshow(output.squeeze(0).detach().numpy().argmax(0))
plt.show()
```
Yet the output is always background. What did I do wrong?

**Comment by wkentaro** [comment]
Check here for how to transform.
https://github.com/wkentaro/pytorch-fcn/blob/eecae876130930d1261b1ae0113787ddf798abcd/torchfcn/datasets/voc.py#L80-L87
